tree_test.go

```diff
@@ -1324,14 +1324,107 @@ func TestLoadVersionForOverwriting(t *testing.T) {
 	_, _, err = tree.SaveVersion()
 	require.NoError(err, "SaveVersion should not fail, write the same value")
 
-	//The tree version now is 52 which is equal to latest version.
-	//Now any key value can be written into the tree
+	// The tree version now is 52 which is equal to latest version.
+	// Now any key value can be written into the tree
 	tree.Set([]byte("key any value"), []byte("value any value"))
 	_, _, err = tree.SaveVersion()
 	require.NoError(err, "SaveVersion should not fail.")
 }
 
-//////////////////////////// BENCHMARKS ///////////////////////////////////////
+func TestDeleteVersionsCompare(t *testing.T) {
```

Comment:
For comparison, you can see my TestDeleteVersionsCompare, which I wrote for debugging, and maybe it should be deleted already. Here is its result:
```
=== RUN   TestDeleteVersionsCompare
    tree_test.go:1365: DeleteVersionsFromInterval database.size 1614
    tree_test.go:1395: DeleteVersion database.size 1614
    tree_test.go:1423: DeleteVersions database.size 1618
--- PASS: TestDeleteVersionsCompare (0.04s)
```